constructor/header.sh

```diff
@@ -537,6 +537,9 @@ if [ "$BATCH" = "0" ]; then
 #if initialize_conda is True and initialize_by_default is False
     DEFAULT=no
 #endif
+#if initialize_conda is False
+    DEFAULT=no
+#endif
```

Comment:
This is not fixing it as initialize_by_default has the None value and `initialize_conda is True`... see this debug build https://github.com/dbast/constructor/actions/runs/4030769478/jobs/6929654326#step:12:167